src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -31,13 +31,15 @@ const Body = BaseForm.extend(
       const sendText = ( this.model.get('primaryMode') === 'sms' )
         ? loc('oie.phone.verify.sms.sendText', 'login')
         : loc('oie.phone.verify.call.sendText', 'login');
+      const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
       const extraCssClasses =
         this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
           'strong no-translate' : '';
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${sendText}
         <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>${this.model.escape('phoneNumber')}</span>
-      </div>`);
+      </div>
+      <div class="okta-form-subtitle" data-se="o-form-explain">${carrierChargesText}</div>`);
```

Comment:
instead we can put this inside existing `div` (okta-form-subtitle)  instead of creating a new one..

Reply:
I put the new content inside `okta-form-subtitle ` div but I still have to create new div to create a spacing between the two section.

Reply:
that would be fine, at least the whole content is part of just one container and that is `okta-form-subtitle`

Reply:
updated both locations